linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faildelay_delay/ansible/ubuntu.yml

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Ensure pam_faildelay.so is properly configured in /etc/pam.d/common-auth
```

Comment:
Please, ensure the [Project Style Guides](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)